code/controllers/subsystem/blackbox.dm

```diff
@@ -32,7 +32,7 @@ SUBSYSTEM_DEF(blackbox)
 
 	if(CONFIG_GET(flag/use_exp_tracking))
 		if((triggertime < 0) || (world.time > (triggertime +3000)))	//subsystem fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check

```

Comment:
time value would need to be changed as well. (or removed since it would have less of an effect)